### PR TITLE
Retro deputy

### PR DESCRIPTION
**Pull request by victorwuky**

### CODE REVIEW

**Comment by victorwuky** [comment]
Requesting review from @maiamcc and @landism since y'all hosted the last two retros.

File: retros/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Retros
+
+We have a retro at least once a month. The retro deputy is responsible for scheduling and facilitating the retro meeting. The retro deputy is responsible for finding a retro deputy for the following month and reminding them about retro deputy duties.
```
Comment (maiamcc) [inline retros/README.md:3]: As per our discussion at the handbook meeting: can you add a bit here about WHY (i.e. the goal of retros and why we have them at this cadence) so the next person to see this doc knows why it's here and how best to change it?

Comment (victorwuky) [reply retros/README.md]: @maiamcc : Can you please review again?

**Comment by maiamcc** [comment]
lgtm!

File: retros/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- How we plan for the future
+- How we work on internal systems
+
+## Retro deputy schedule
+The retro deputy is responsible for scheduling and facilitating the retro meeting. The retro deputy is responsible for finding a retro deputy for the following month and reminding them about retro deputy duties.
+
+| Month | Deputy |
+| --- | --- |
+| 2020 July | Matt, Maia |
+| 2020 Aug | |
```
Comment (nicks) [inline retros/README.md:16]: han

Comment (victorwuky) [reply retros/README.md:16]:
```suggestion
| 2020 Aug | Han |
```

cc @hyu

**Pull request merged**